server/modules/slack/slackIntegration.js

```diff
@@ -160,3 +168,27 @@ export const addToChannel = async (fellowEmail, channel) => addOrRemove(fellowEm
  * @returns {Object} - The result after a user has been removed from a channel
  */
 export const removeFromChannel = async (fellowEmail, channel) => addOrRemove(fellowEmail, channel, 'kick');
+
+/**
+ * @function onboardingAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} channel - the channel id
+ * @param {string} partnerId - the prtner id
+ *
+ * @returns {Object} - With a success message
+ */
+
+export const onboardingAutomation = async (fellowEmail, channel, partnerId) => {
+  try {
+    removeFromChannel(fellowEmail, channel);
+    const channelDetails = await createPartnerChannels(partnerId);
+    const partnerGeneralChannelId = channelDetails.generalChannel.id;
+    addToChannel(fellowEmail, partnerGeneralChannelId);
+    return { status: 'ok', message: 'onboarding automation was successful' };
```

Comment:
A helper function has been created to aid the sending of response(either error or success), please make use of it here.
You can check `server/helper/response.js` to see how to use it.